<launch>
    <node name="zbar_decoder_node" pkg="zbar_decoder" type="zbar_decoder_node" respawn="false" output="screen" >
      <!-- <remap from="/tag" to="qr_tag" /> -->
      <param name="tag_scale_factor" value="0.5" />
      <param name="debounce_size" value="3" />
      <param name="threshold" value="190" />
    </node>
</launch>
